### PR TITLE
v0.3.1

### PR DESCRIPTION
**Pull request by michaelshobbs**
Not sure if I'm doing this right but here goes. Also, someone will want to merge this into master I suppose?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,21 @@ All notable changes to this project will be documented in this file.
...
 
+## [0.3.1] - 2015-07-09
+### Fixed
+- Fixed directory permission for custom buildpacks
+
+### Added
+- Added ability to build in docker container
+
+### Removed
+- Remove testing play-v1 from play buildpack
```
Comment (josegonzalez) [inline CHANGELOG.md]: This is a breaking change, so probably worth making 0.4.0, no?

Comment (michaelshobbs) [inline CHANGELOG.md]: It is?

Comment (michaelshobbs) [inline CHANGELOG.md]: We just removed the test not the support. The scala buildpack handles play-v1 as I understand it anyway.

Comment (josegonzalez) [inline CHANGELOG.md]: Oks.

**Pull request merged**